cms/tests/test_user_tasks.py

```diff
@@ -0,0 +1,95 @@
+"""
+Unit tests for integration of the django-user-tasks app and its REST API.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from uuid import uuid4
+
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import override_settings
+
+from rest_framework.test import APITestCase
+from user_tasks.models import UserTaskArtifact, UserTaskStatus
+from user_tasks.serializers import ArtifactSerializer, StatusSerializer
+
+
+# Helper functions for stuff that pylint complains about without disable comments
+
+def _context(response):
+    """
+    Get a context dictionary for a serializer appropriate for the given response.
+    """
+    return {'request': response.wsgi_request}  # pylint: disable=no-member
+
+
+def _data(response):
+    """
+    Get the serialized data dictionary from the given REST API test response.
+    """
+    return response.data  # pylint: disable=no-member
+
+
+@override_settings(BROKER_URL='memory://localhost/')
+class TestUserTasks(APITestCase):
+    """
+    Tests of the django-user-tasks REST API endpoints.
+
+    Detailed tests of the default authorization rules are in the django-user-tasks code.
+    These tests just verify that the API is exposed and functioning.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = User.objects.create_user('test_user', 'test@example.com', 'password')
+        cls.status = UserTaskStatus.objects.create(
+            user=cls.user, task_id=str(uuid4()), task_class='test_rest_api.sample_task', name='SampleTask 2',
+            total_steps=5)
+        cls.artifact = UserTaskArtifact.objects.create(status=cls.status, text='Lorem ipsum')
+
+    def setUp(self):
+        super(TestUserTasks, self).setUp()
+        self.status.refresh_from_db()
+        self.client.force_authenticate(self.user)  # pylint: disable=no-member
+
+    def test_artifact_detail(self):
+        """Users should be able to access artifacts for tasks they triggered."""
```

Comment:
edx style guide asks that all comments are multiline: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html